Library/AppEnvironment.swift

```diff
@@ -16,15 +17,15 @@ public struct AppEnvironment {
   /**
    A global stack of environments.
    */
-  private static var stack: [Environment] = [Environment()]
+  fileprivate static var stack: [Environment] = [Environment()]
```

Comment:
the swift 3 migrator was waaaaay too aggressive with changing `private` to `fileprivate` so i wanna take a separate pass at renaming these to `private`...